guides/common/modules/proc_enrolling-project-server-in-freeipa-domain.adoc

```diff
@@ -0,0 +1,116 @@
+[id="enrolling-project-server-in-freeipa-domain_{context}"]
+= Enrolling {ProjectServer} in a {FreeIPA} domain
+
+Create a host entry for your {ProjectServer} system in the {FreeIPA} LDAP and configure the system to be a client in your {FreeIPA} domain.
+
+.Prerequisites
+
+* An existing {FreeIPA} server.
+
+.Procedure
+
+. On the {FreeIPA} server:
+.. Create a host entry for the {ProjectServer} system.
+ifdef::satellite[]
++
+For more information, see link:{RHELDocsBaseURL}8/html/configuring_and_managing_identity_management/index[{RHEL}{nbsp}8 Configuring and managing Identity Management] or link:{RHELDocsBaseURL}9/html/configuring_and_managing_identity_management/index[{RHEL}{nbsp}9 Managing IdM users, groups, hosts, and access control rules].
+endif::[]
+.. Create an entry for the HTTP service for {ProjectServer}.
+ifdef::satellite[]
++
+For more information on creating a service entry in {FreeIPA}, see
+link:{RHELDocsBaseURL}8/html-single/managing_idm_users_groups_hosts_and_access_control_rules/index[{RHEL}{nbsp}8 Managing IdM users, groups, hosts, and access control rules] or link:{RHELDocsBaseURL}9/html-single/managing_idm_users_groups_hosts_and_access_control_rules/index#[{RHEL}{nbsp}9 Managing IdM users, groups, hosts, and access control rules].
+endif::[]
+. On {ProjectServer}:
+.. Configure the {ProjectServer} system as client in the {FreeIPA} domain.
+This includes ensuring that the system meets the necessary prerequisites, installing the necessary packages, and running the {FreeIPA} client installation utility.
++
+[NOTE]
+====
+To install the {FreeIPA} client package, use the `{foreman-installer}` utility.
+====
+ifdef::satellite[]
++
+For more information, see link:{RHELDocsBaseURL}/8/html-single/installing_identity_management/index#assembly_installing-an-idm-client_installing-identity-management[{RHEL}{nbsp}8 Installing Identity Management] or link:{RHELDocsBaseURL}/9/html-single/installing_identity_management/index#assembly_installing-an-idm-client_installing-identity-management[{RHEL}{nbsp}9 Installing Identity Management].
+endif::[]
+ifdef::foreman-deb[]
++
+. Ensure that the hostname is set to the fully qualified domain name (FQDN); the short name is not sufficient:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# hostname
+{foreman-example-com}
+----
++
+Otherwise, `{foreman-installer}` cannot generate the right principal name that is needed to join the realm.
+endif::[]
+
+.Verification
+
+* On your {ProjectServer}, check that you are able to resolve a user defined on the {FreeIPA} server.
+For example, to check the `admin` user that {FreeIPA} creates by default:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+$ id admin
```

Comment:
@domiborges Should we give more details about what the expected output is? For example, tell users to run `id admin` on both the Foreman server/IPA client and the IPA server and make sure that the IDs match?